lua/possession/plugins/delete_buffers.lua

```diff
@@ -0,0 +1,16 @@
+local M = {}
+
+function M.before_load(_, _, plugin_data)
+    local current_buffer = vim.api.nvim_get_current_buf()
+    vim.pretty_print(vim.api.nvim_list_bufs())
```

Comment:
You probably forgot to delete this debug print? Or is the PR still WIP (as I see you already force-pushed a few times)?

Reply:
Now this bug has been fixed